src/main/java/com/fasterxml/jackson/databind/cfg/MapperConfig.java

```diff
@@ -596,6 +596,15 @@ public abstract VisibilityChecker<?> getDefaultVisibilityChecker(Class<?> baseTy
      */
     public abstract Boolean getDefaultMergeable(Class<?> baseType);
 
+    /**
+     * Accessor for polymorphic type handling settings to use for given base type.
+     * 
+     * @since 2.16
+     */
+    public JsonTypeInfo.Value getDefaultPolymorphicTypeHandling(Class<?> baseType) {
```

Comment:
Can be left as abstract (this is abstract class & expectation is that all subtypes come from databind itself)